website/docs/r/integration_service_environment.markdown

```diff
@@ -0,0 +1,136 @@
+---
+subcategory: "Logic App"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_integration_service_environment"
+description: |-
+  Manages a private and isolated Logic Apps instance within Azure virtual network.
+---
+
+# azurerm_integration_service_environment
+
+Manages a private and isolated Logic Apps instance within Azure virtual network.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "exampleRG1"
+  location = "westeurope"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet1"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  address_space       = ["10.0.0.0/22"]
+}
+
+resource "azurerm_subnet" "isesubnet1" {
+  name                 = "isesubnet1"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.0/26"]
+
+  delegation {
+    name = "integrationServiceEnvironments"
+    service_delegation {
+      name = "Microsoft.Logic/integrationServiceEnvironments"
+    }
+  }
+}
+
+resource "azurerm_subnet" "isesubnet2" {
+  name                 = "isesubnet2"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.64/26"]
+}
+
+resource "azurerm_subnet" "isesubnet3" {
+  name                 = "isesubnet3"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.128/26"]
+}
+
+resource "azurerm_subnet" "isesubnet4" {
+  name                 = "isesubnet4"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.192/26"]
+}
+
+resource "azurerm_integration_service_environment" "example" {
+
+  name                 = "example-ise"
+  location             = azurerm_resource_group.example.location
+  resource_group_name  = azurerm_resource_group.example.name
+  sku_name             = "Developer"
+  capacity             = 0
```

Comment:
```suggestion
  sku_name             = "Developer_0"
```

Reply:
Not sure if I follow here. Are you proposing to introduce new artificial SKU name? Would not be this kind of confusing? We have validation for SKU Developer and capacity 0 so we can catch bad input early - if this is the concern.

Reply:
Thanks, I changed it.